web/docs/setup-guides/deploy-a-node.md

```diff
@@ -63,6 +63,18 @@ dpkg -i tenzir-static-amd64-linux.deb
 
 [tenzir-debian-package]: https://github.com/tenzir/tenzir/releases/latest/download/tenzir-static-amd64-linux.deb
 
+</TabItem>
+<TabItem value="rpm_based" label="RPM based (RedHat / OpenSUSE / ...)">
+
+Download the latest [RPM package][tenzir-rpm-package] and install it via
+`rpm`:
+
+```bash
+yum -y localinstall tenzir-static-amd64-linux.rpm
```

Comment:
With `dnf` it should be `sudo dnf localinstall tenzir-static-amd64-linux.rpm` if I Googled the right thing. I think we should document that as well.

Should we add the sudo for `yum` as well?

Reply:
We also didn't add the `sudo` in the other tabs, and I think either way is as good as the other, as long as it is consistent.

Reply:
Makes sense.